utils/path/src/data_dir.rs

```diff
@@ -42,6 +42,6 @@ impl FromStr for DataDir {
 
 impl Display for DataDir {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
-        write!(f, "{:?}", self.0)
+        write!(f, "{}", self.0.to_string_lossy())
```

Comment:
This is required, because otherwise flexi logger tries to create a path like `"/home/user/.local/share/yagna"/yagna.log` and those extra quotes are causing problems.

Reply:
This comment was very helpful for me! Could you put this comment in source also?

Reply:
yes